### PR TITLE
custodianship is a multi-value attribute, not a boolean

### PR DESCRIPTION
**Pull request by asaunier**
Fixes https://github.com/c2corg/v6_api/issues/207

### CODE REVIEW

File: c2corg_api/scripts/migration/documents/waypoints/huts.py [hunk 1]
```diff
@@ -94,6 +95,16 @@ def get_document_locale(self, document_in, version):
             access_period=document_in.staffed_period
         )
 
+    def set_custodianship(self, document_in):
```
Comment (tsauerwein) [inline c2corg_api/scripts/migration/documents/waypoints/huts.py]: `get_custodianship`?

File: c2corg_api/models/waypoint.py [hunk 2]
```diff
@@ -139,7 +139,7 @@ class _WaypointMixin(object):
     phone_custodian = Column(String(50))
 
     # gardiennage (gite, camp_site, hut, abri, bivouac, base_camp)
-    custodianship = Column(Boolean)
+    custodianship = Column(enums.custodianship_type)
```
Comment (tsauerwein) [inline c2corg_api/models/waypoint.py]: With 'multi-value' you don't mean you can select multiple values? You would have to use `Column(ArrayOfEnum(enums.custodianship_type))` in that case.

Comment (asaunier) [inline c2corg_api/models/waypoint.py]: I meant it is a field with more value than yes/no. But only one value at a time can be selected. Do you think I should use another word than "multi-value"?

File: c2corg_api/scripts/migration/documents/waypoints/huts.py (hunk 1 above)
Comment (asaunier) [inline c2corg_api/scripts/migration/documents/waypoints/huts.py]: The goal of the function is to set the value of the ``custodianship`` attribute, isn't it?

File: c2corg_api/models/waypoint.py (hunk 2 above)
Comment (tsauerwein) [inline c2corg_api/models/waypoint.py]: Ok. No, you don't have to change the commit message for me. I just wanted to make sure that we are talking about the same thing.

File: c2corg_api/scripts/migration/documents/waypoints/huts.py (hunk 1 above)
Comment (tsauerwein) [inline c2corg_api/scripts/migration/documents/waypoints/huts.py]: You are using the return value of this function to set the attribute. But this function does not set an attribute.

Comment (asaunier) [inline c2corg_api/scripts/migration/documents/waypoints/huts.py]: Right. I have pushed a fixup.

**Comment by asaunier** [comment]
The spaces within the custodianship values seem to cause no problem as far as I can see.
But to be consistent with the other attributes I have added them. By the way I have replaced "stub" by "draft" in the quality types since the client prefers this term.

OK for merging the PR?

**Comment by tsauerwein** [comment]
:+1:

**Pull request merged**